src/Avatar/utils.ts

```diff
@@ -0,0 +1,28 @@
+function hashCode(string: string) {
+  let hash = 0;
+  // eslint-disable-next-line no-plusplus
+  for (let i = 0; i < string.length; i++) {
+    // eslint-disable-next-line no-bitwise
+    hash = string.charCodeAt(i) + ((hash << 5) - hash);
+  }
+  return hash;
+}
+
+function getHslColor(hash: number) {
+  const h = range(hash, 0, 360);
+  const s = range(hash, 75, 80);
+  const l = range(hash, 40, 70);
+  const opacity = '0.9';
+
+  return `hsla(${h}, ${s}%, ${l}%,${opacity})`;
+}
+
+function range(hash: number, min: any, max: any) {
+  const diff = max - min;
+  const x = ((hash % diff) + diff) % diff;
+  return x + min;
+}
+
+export function stringToHexColor(string: string): string {
```

Comment:
Needs testing, preferably some kind of property based test. Things that need to be true about this function:
- Various inputs always output a correctly formatted hex code
- The hex code of various inputs differs
- It is pure

Reply:
added tests